easybuild/easyconfigs/k/Kaiju/Kaiju-1.7.2-makefile.patch

```diff
@@ -0,0 +1,32 @@
+diff kaiju-1.7.2/src/bwt/Makefile{.orig,} -ru
```

Comment:
@Darkless012 For patch files we try to always add a short description above + mention author.

In this case, this probably makes most sense (see also #4349):

```
avoid hardcoding compiler command and flags
author: wpoely86@gmail.com, updated by Pavel Grochal (INUITS)
--- kaiju-1.7.2/src/bwt/Makefile.orig	2019-07-12 19:32:23.000000000 +0200
...
```

Note that you'll also need to update the SHA256 checksum for the patch file in the easyconfig.